src/components/MediaPlayer.tsx

```diff
@@ -15,10 +15,11 @@ interface IProps extends IStateProps {
   srcMediaId: string;
   requestPlay: boolean;
   onEnded: () => void;
+  onError: (msg: string) => void;
 }
 
 export function MediaPlayer(props: IProps) {
-  const { auth, srcMediaId, requestPlay, onEnded, ts } = props;
+  const { auth, srcMediaId, requestPlay, onEnded, onError, ts } = props;
```

Comment:
should we just get the reporter from the global and handle the error reporting here completely? I don't think you do anything when you have an error. Alternatively, the responsibility for reporting and handling of the error should be with the caller... At least in this case, since the caller won't do anything but report it, it is probably fine to handle it here although that may make the code less reusable.